README.md

```diff
@@ -835,6 +835,14 @@ _Consul Enterprise Only (requires that CONSUL_ENTERPRISE is set to true)_
  - If the default config template does not suit your needs, you can replace it with your own.
  - Default value: `templates/config.json.j2`.
 
+### `consul_rolling_restart`
+ - Restarts consul node one by one to avoid service interruption on existing cluster (Linux platforms only).
```

Comment:
```suggestion
 - Restarts consul node one by one to avoid service interruption on existing cluster (Unix platforms only).
```

Since technically it should work on other platforms, and the tasks do state "Unix platforms only".